indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -520,18 +520,17 @@ def can_comment(self, user, check_state=False):
             return True
         return self.can_judge(user) or self.can_convene(user) or self.can_review(user)
 
-    def can_convene(self, user):
+    def can_convene(self, user, check_state=False):
```

Comment:
With the suggestion above we no longer use `check_state=True` - and when having a quick look I didn't spot other places where we care about the state, so the change here could be reverted.

Thinking about this again, for judging/reviewing check_state makes sense,  but "convening" (is that even a word? :D) isn't really a "modifying" action so checking the state for it doesn't make that much sense in the end...

Reply:
Agree, probably not relevant anymore